contracts/Dispenser.sol

```diff
@@ -267,13 +267,18 @@ contract Dispenser {
     event TreasuryUpdated(address indexed treasury);
```

Comment:
Hm, are these changes not already on main?

Reply:
Correct, wanted to just merge in order to get rid of git leaks errors, but forgot where the branch was initiated and that it is pretty far in the sequence of commits.